create-cluster.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+
+# See example.env.create-cluster for the environment variables
+# which must be set before running this script.
+#
+# Usage:
+#
+#   ./create-cluster.sh [cluster-name]
+#
+
+readonly MAX_CLUSTER_NAME_LENGTH=12
+readonly CLUSTER_SUFFIX=cloud-platform.service.justice.gov.uk
+# TODO: use the right dns flush command, depending on the architecture of the local machine
+readonly DNS_FLUSH_COMMAND='sudo killall -HUP mDNSResponder' # Mac OSX Mojave
+
+set -euo pipefail
+
+main() {
+  local readonly cluster_name=$1
+
+  check_prerequisites ${cluster_name}
+
+  git-crypt unlock
+  get_sudo
+
+  create_cluster ${cluster_name}
+  run_kops ${cluster_name}
+  install_components ${cluster_name}
+
+  kubectl cluster-info
+}
+
+create_cluster() {
+  local readonly cluster_name=$1
+  (
+    cd terraform/cloud-platform
+    switch_terraform_workspace ${cluster_name}
+    terraform apply -auto-approve
+  )
+}
+
+run_kops() {
+  local readonly cluster_name=$1
+
+  kops create -f kops/${cluster_name}.yaml
+
+  # This is a throwaway SSH key which we never need again.
+  rm -rf /tmp/${cluster_name}*
```

Comment:
May I suggest simply `rm -f /tmp/${cluster_name}` to avoid any happy accidents? :)

Reply:
There will be a key and a key.pub file. I'll change it to specify those explicitly.

Reply:
(I was more concerned about the `-r` flag, not the wildcard)

Reply:
Yeah, that was a mistake - just muscle memory!